junos/resource_interface_physical.go

```diff
@@ -23,6 +23,7 @@ type interfacePhysicalOptions struct {
 	description string
 	v8023ad     string
 	vlanMembers []string
+	Esi         []map[string]interface{}
```

Comment:
useless export variable, name need to start with lowercase
```suggestion
	esi         []map[string]interface{}
```